### ISSUE TITLE
Keep the "vicinity" of the current image when going back to the timeline of the web interface

### ISSUE THREAD

**Issue by steinarb** [pixelfed/pixelfed#5181]
It would be nice if, in the web interface, when going back to the timeline from an image I would come back to the "vicinity" of the image.

If I've e.g. opened an image from July 2022, it would be nice if the timeline would open scrolled to July 2022.

**Comment by HelmutsCab** [comment]
You are right, that's annoying. The position of the TL should remain the same and not jump back to the beginning every time you click on a picture. That would be great.

**Comment by steinarb** [comment]
Pity. I think this would have been useful.

Right now as a workaround I am opening images in new tabs and deleting them to go back to the current place in the timeline (which may be years back).
